airflow/providers/presto/hooks/presto.py

```diff
@@ -112,8 +112,20 @@ def get_isolation_level(self) -> Any:
     def _strip_sql(sql: str) -> str:
         return sql.strip().rstrip(';')
 
+    @overload
+    def get_records(self, sql: str = "", parameters: Optional[dict] = None):
+        """Get a set of records from Presto
+        :param sql: SQL statement to be executed.
+        :param parameters: The parameters to render the SQL query with.
+        """
+        ...
+
+    @overload
     def get_records(self, sql: str = "", parameters: Optional[dict] = None, hql: str = ""):
-        """Get a set of records from Presto"""
+        ...
```

Comment:
What does this overload do?

Reply:
Static checks were failing on local(MyPy) - Single overload definition, multiple required.

And the workaround I found was to have two different signatures in two @overload signatures. And the actual function header for the implementation should be empty.